packages/marble_cli/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: marble_cli
+description: CLI tools used to build and run Marble sites
+version: 0.0.1
+publish_to: none
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+
+dependencies:
+  mason_logger: ^0.1.0-dev.12
+  path: ^1.8.2
+  pub_updater: ^0.2.1
+
+dev_dependencies:
+  build_runner: ^2.0.0
+  build_verify: ^3.0.0
+  build_version: ^2.0.0
+  mocktail: ^0.3.0
+  test: ^1.19.2
+  universal_io: ^2.0.4
```

Comment:
```suggestion
```

Reply:
My tests doesn't compile without that.

Btw I had a note to talk about you regarding this...There is a test generated by the `dart_cli` template that uses this package. And the pubspec doesn't includes it. So I just added it.

Is the template incorrect? Or did I miss something?

Reply:
It was a mistake and we fixed it in the template but still need to publish a new release of very_good_cli

Reply:
Alright, I will check how to fix the test to not use that package.

Reply:
Solved, thanks!!!!